src/utils/logger.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var PREFIX = '[algoliasearch-helper]';
+
+function warn(message) {
+  console.warn(PREFIX + ' ' + message);
```

Comment:
I don't see an eslint error here, why not using console.warn directly in the setState?

Reply:
https://github.com/algolia/algoliasearch-helper-js/pull/802/commits/a1c3d3deb520d15046258761a64d28be93ff85da